deployment/ansible/roles/district-builder.app-server/templates/env.conf.j2

```diff
@@ -0,0 +1,23 @@
+DEBUG=False
```

Comment:
We should remove probably remove this entirely (I believe it becomes the truthy string `"False"` so `DEBUG` will evaluate to `True` in a boolean context)